test/functional/workgroup/CMakeLists.txt

```diff
@@ -13,16 +13,29 @@
 # hipcc_cmake_linker_helper because it expects the path to hipcc as the first
 # argument
 #
-macro( buildunitworkgrouptest TESTNAME SUBTESTNAMES BACKENDS )
+macro( buildfunctionalworkgrouptest TESTNAME SUBTESTNAMES BACKENDS )
```

Comment:
@rhornung67 This is probably the intended name for this macro, right? When I was adding the "Known-Hip-Failure" tag to these tests, I noticed this. Let me know if I should keep this

Reply:
uhhh. I don't know about that. @MrBurmark what should we do here?

Reply:
There are two macros with the same name, looks like a copy paste. This macro is in the functional tests, so I think the name Kristi proposed makes sense.

This caused issues since it wasn't clear where to perform the "known failure" tagging.

Reply:
Ya this is a copy paste, that makes sense.